tests/integration/block_createBlock_api_test.go

```diff
@@ -37,7 +37,16 @@ var (
 )
 
 func TestBlockImageCreateSuite(t *testing.T) {
-	suite.Run(t, new(BlockImageCreateSuite))
+	s := new(BlockImageCreateSuite)
+	defer func(s *BlockImageCreateSuite) {
+		if r := recover(); r != nil {
```

Comment:
this same function is repeated in multiple places now.  would it be possible to refactor so the function is only defined in a single place?  I think that would mean defining an interface for the common function called across all the suite types in this function, which is `TearDownSuite()`.  Then the single function that does this work would accept an argument of that interface type.

Reply:
I cleaned up the code as you suggested @jbw976. Using a interface to to the Teardown/cleanup now.And using a single fumction to to the recover.